pages/whats-new/tinacms.tsx

```diff
@@ -0,0 +1,66 @@
+import { Layout } from 'components/layout';
+import client from 'tina/__generated__/client';
+import { TinaMarkdown } from 'tinacms/dist/rich-text';
+import { FaPlus, FaMinus } from 'react-icons/fa';
+import { useState } from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
+
+export const versionComponents = {
+  h1: (props) => <h1 className="text-lg font-bold mt-8 mb-4" {...props} />,
+  h2: (props) => <h2 className="text-lg font-semibold mt-6 mb-4" {...props} />,
+  h3: (props) => <h3 className="text-lg font-semibold mt-5 mb-3" {...props} />,
+  h4: (props) => <h4 className="text-lg font-medium mt-4 mb-3" {...props} />,
+  h5: (props) => <h5 className="text-lg font-medium mt-3 mb-2" {...props} />,
+  h6: (props) => <h6 className="text-base font-medium mt-2 mb-2" {...props} />,
+  ul: (props) => <ul className="list-disc ml-5 mb-4" {...props} />,
+  ol: (props) => <ol className="list-decimal ml-5 mb-4" {...props} />,
+  li: (props) => <li className="mb-2" {...props} />
+};
```

Comment:
we are exporting this on both pages... can it be moved somewhere generic?

Reply:
Done, moved to component/styles/WhatsNewVisualComponents.tsx